packages/service/README.md

```diff
@@ -0,0 +1,111 @@
+# @loopback/service
+
+This module provides a common set of interfaces for interacting with service
+oriented backends such as REST APIs, SOAP Web Services, and gRPC microservices.
+
+## Installation
+
+```
+$ npm install --save @loopback/service
+```
+
+## Usage
+
+### Define a data source for the service backend
+
+```ts
+import {DataSourceConstructor, juggler} from '@loopback/service';
+
+const ds: juggler.DataSource = new DataSourceConstructor({
+  name: 'GoogleMapGeoCode',
+  connector: 'rest',
+  options: {
+    headers: {
+      'accept': 'application/json',
+      'content-type': 'application/json'
+    }
+  },
+  operations: [
+    {
+      template: {
+        method: 'GET',
+        url: 'http://maps.googleapis.com/maps/api/geocode/{format=json}',
+        query: {
+          address: '{street},{city},{zipcode}',
+          sensor: '{sensor=false}'
+        },
+        responsePath: '$.results[0].geometry.location[0]'
+      },
+      functions: {
+        geocode: ['street', 'city', 'zipcode']
+      }
+    }
+  ]
+});
+```
+
+### Bind data sources to the context
+
+```ts
+import {Context} from '@loopback/context';
+
+const context = new Context();
+context.bind('dataSources.geoService').to(ds);
+```
+
+**NOTE**: Once we start to support declarative datasources with `@loopback/boot`,
+the datasource configuration files can be dropped into `src/datasources` to be
+discovered and bound automatically.
```

Comment:
Please add a link to the GitHub issue tracking the task of adding support for data-sources to `@loopback/boot`.